apps/poker-roll/src/main.tsx

```diff
@@ -1,13 +1,41 @@
 import { StrictMode } from 'react';
 import * as ReactDOM from 'react-dom/client';
+import {
+  createBrowserRouter,
+  RouterProvider,
+} from "react-router-dom";
```

Comment:
Doesn't look like `react-router-dom` got added to your `package.json`. Did you run `npm install react-router-dom` from the repo root?

Reply:
npm install has been run now.